cmd/root.go

```diff
@@ -267,6 +273,8 @@ func init() {
 	_ = rootCmd.PersistentFlags().MarkHidden("await-continue")
 	_ = rootCmd.PersistentFlags().MarkHidden("await-open")
 	_ = rootCmd.PersistentFlags().MarkHidden("debug-skip-files")
+
+	rootCmd.PersistentFlags().StringVar(&azcopyFromToFlag, "from-to", "", "Used to check if from-to is BlobPipe type then avoid version check as it appends version update info log which causes failure in download.")
```

Comment:
Will this create a new flag as the from-to will be there already defined in some other file.

Reply:
updated. using GetString() to retrieve the flag value